README.md

```diff
@@ -47,9 +47,9 @@ Files matching this pattern will be included in the manifest.
 
 _Default:_ `"**/*.{js,css,png,gif,ico,jpg,map,xml,txt,svg,swf,eot,ttf,woff,woff2}"`
 
-### ignoreFilePattern
+### fileIgnorePattern
 
-Files matching this pattern will _not_ be included in the manifest even if they match filePattern.
+Files matching this pattern will _not_ be included in the manifest even if they match filePattern. And thus will be allways deployed.
```

Comment:
Great stuff @bortevik . Give me a shout if you have any questions at all. You can also get in contact with myself and the rest of the core team on the ember community slack in the #ec-deploy channel.